Sprint-3/revise/implement/is-prime.test.js

```diff
@@ -1,3 +1,46 @@
 // Given a positive integer num,
 // When the isPrime function is called with num as input,
 // Then it should check if the num is prime
+______________________________________SOLUTION_______________________________________________________
+
+// Function to check if a number is prime
+function isPrime(num) {
+    // Special case: if num is less than 2, it's not prime
+    if (num < 2) {
+      return false;
+    }
+  
+    // Loop from 2 up to the square root of num
+    for (let i = 2; i <= Math.sqrt(num); i++) {
```

Comment:
For better performance, you could 
- check number 2 separately and only checks odd numbers in the loop, and
- assign `Math.sqrt(num)` to a variable first and refer to the the variable repeatedly in the loop.